app/views/admin/admin_customers/index.html.erb

```diff
@@ -8,6 +8,11 @@
     <%= f.text_field :address_cont, placeholder: t("admin_customer.placeholder_address"), class: "form-control" %>
     <%= f.submit t("admin_customer.button_search"), class: "form-control" %>
   <% end %>
+  <div class="trash_soft_delete">
+    <%= link_to "?deleted_at=1", class: "btn btn-warning" do %>
```

Comment:
đường dẫn như thế này luôn har em? Nhìn kỳ kỳ

Reply:
À cái này là cái param em truyền lên để kiểm tra (phân biệt) tập các bản ghi đã xóa và chưa xóa thôi ạ, cái param đặt là gì cũng được ạ. Cái param này hoàn toàn không cố định ạ